modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/policy-navigator-details/history-info/index.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import PanelComponent from '../../../../../../common/components/panel';
+import {InfoPanelType} from '../types';
+
+type HistoryInfoType = {
+	arraySortedByDate: InfoPanelType[];
+	setShowPanel: (panel: boolean[]) => void;
+	showPanel: boolean[];
+};
+
+const HistoryInfo = ({
+	arraySortedByDate,
```

Comment:
This variable `arraySortedByDate`, what is this?

array of what? I know in line 19 you used InfoPanelType, that way you could put a clear name, see:

* https://github.com/ryanmcdermott/clean-code-javascript#use-searchable-names
* https://github.com/ryanmcdermott/clean-code-javascript#use-meaningful-and-pronounceable-variable-names

Try this: 

```suggestion
	InfoPanels,
```

or

```suggestion
	items,
```

Reply:
InfoPanels is Good! 
Thx!